.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "webpack",
+  "globals": {
```

Comment:
I think we don’t need this anymore, since webpack preset includes `jest` environment.